### PR TITLE
German prompts

### PR DESCRIPTION
**Pull request by ischender**
Moving most of the functions involving openAI to the German code, and translating prompts.

### CODE REVIEW

File: src/langcheck/metrics/de/reference_free_text_quality.py
Comment (yosukehigashi) [inline src/langcheck/metrics/de/reference_free_text_quality.py:96]: This is done!

File: src/langcheck/metrics/de/reference_free_text_quality.py
Comment (yosukehigashi) [inline src/langcheck/metrics/de/reference_free_text_quality.py:227]: Fluency also has an OpenAI version, so for that one, we can pass in the German text directly.

File: src/langcheck/metrics/de/source_based_text_quality.py
Comment (yosukehigashi) [inline src/langcheck/metrics/de/source_based_text_quality.py:85]: This is done!

File: src/langcheck/metrics/de/reference_free_text_quality.py
Comment (yosukehigashi) [inline src/langcheck/metrics/de/reference_free_text_quality.py:1]: We recently added a new metric called `answer_relevance` in the English and Japanese versions, which just has an OpenAI version. So if you could also translate the prompt to german, that would be great!

**Review by yosukehigashi — COMMENTED** [review]
Thanks @ischender!

File: src/langcheck/metrics/de/reference_free_text_quality.py
Comment (ischender) [reply src/langcheck/metrics/de/reference_free_text_quality.py:96]: True, to be honest, I did not investigate the improvements...

File: src/langcheck/metrics/de/reference_free_text_quality.py
Comment (ischender) [reply src/langcheck/metrics/de/reference_free_text_quality.py:227]: ah yes, I missed that, adding that too...

**Review by yosukehigashi — COMMENTED** [review]
Thanks! Just a couple of docstring comments, but otherwise LGTM. I'll take a final look after #81 is merged first

File: src/langcheck/metrics/de/reference_free_text_quality.py
Comment (ischender) [reply src/langcheck/metrics/de/reference_free_text_quality.py:236]: good point, done it now

Comment (yosukehigashi) [reply src/langcheck/metrics/de/reference_free_text_quality.py]: Thanks!

**Review by yosukehigashi — APPROVED** [review]
LGTM! Excited to try this out on some German text (even though I won't understand it 😆). I'll merge this once tests pass!

**Pull request merged**
